src/systel.linux.cfg

```diff
@@ -0,0 +1,131 @@
+###
+### See http://wiki.opentelemac.org/doku.php?id=installation_notes_2_beta
+### On how to install telemac
+###
+# _____                              _______________________________
+# ____/ TELEMAC Project Definitions /______________________________/
+#
+[Configurations]
+configs: gnu.dynamic gnu.dynamic.debug
+# _____          ____________________________________
+# ____/ General /___________________________________/
+# Global declaration used for all configurations
+[general]
+language: 2
+modules:  system
+#
+sfx_zip:    gztar
+sfx_lib:    .a
+sfx_obj:    .o
+sfx_mod:    .mod
+sfx_exe:
+#
+#
+val_root:   <root>/examples
+#
+val_rank:   all
+#
+mods_all:   -I <config>
+ar: ar
+lib_flags: cru
+#
+# MPI info
+#
+fc: mpif90
+flags_mpi: -DHAVE_MPI
+# Those are for nag.ad
+incs_mpi: -kind=byte
+libs_mpi: -lmpichf90 -lmpich -lopa -lmpl -lrt -lpthread
+cmd_doxygen:doxygen
+
+#
+# METIS info
+#
+libs_metis: -lmetis
+
+#
+# MED info
+#
+flags_med: -DHAVE_MED
+libs_med: -lm -lmed -lhdf5 -ldl -lstdc++ -lz
+
+#
+# MUMPS info
+#
+flags_mumps: -DHAVE_MUMPS
+libs_so_mumps: -ldmumps -lmumps_common -lpord
+               -lscalapack
+               -lblas
+
+#
+# AED info
+#
+flags_aed: -DHAVE_AED2
+libs_aed: -laed2
+
+#
+# GOTM info
+#
+flags_gotm: -DHAVE_GOTM
+libs_gotm: -lturbulence -lutil
```

Comment:
I don't see this packages in meta.yaml

Reply:
I hadn't included the flags.
I can add GOTM easily, it's already on [conda-forge](https://anaconda.org/conda-forge/gotm)
I saw that you've added a library for AED2, any chance you already have a recipe to pass me?

Reply:
https://github.com/nicogodet/aed2-otm-feedstock

I tried to add gotm to the recipe but gotm is not found during conda build initialisation. If I run conda install gotm, no issue...